src/plugins/blocks/blocks.js

```diff
@@ -237,7 +268,7 @@ function loadBlock(context, event, container) {
   const blockName = getBlockName(blockElement, false);
 
   // Render the preview frame if we haven't already
-  renderFrame(content);
+  renderFrame(context.viewPorts, content);
```

Comment:
Looks great thanks for the contribution @nc-andreashaller!

I'd say let's just pass the whole context in. There are other issues logged where we will need it. Namely https://github.com/adobe/franklin-sidekick-library/issues/88

Reply:
Actually, #88 will actually come in via library metadata. NVM.. we can leave this as is for now.